plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -313,12 +328,83 @@ def _onChanged(self):
 
         self._change_timer.start()
 
-    def _sendSettings(self, profile):
+    def _getTemperatureGcode(self, profile, object_groups):
+        temp_gcode = {
+            "set_bed_temperature": "",
+            "set_bed_temperature_and_wait": "",
+            "set_extruder_temperatures": "",
+            "set_extruder_temperatures_and_wait": ""
+        }
+        if profile.getSettingValue("machine_gcode_flavor") == "UltiGCode":
+            return temp_gcode
+
+        if profile.getSettingValue("machine_heated_bed"):
+            temp = profile.getSettingValue("material_bed_temperature")
+            temp = int(temp) if temp else 0
```

Comment:
There should be no need to explicitly cast to int, that should already be taken care of by the Profile class.